lib/manageiq/api/common/application_controller_mixins/exception_handling.rb

```diff
@@ -0,0 +1,27 @@
+module ManageIQ
+  module API
+    module Common
+      module ApplicationControllerMixins
+        module ExceptionHandling
+          def self.included(other)
+            other.rescue_from(ManageIQ::API::Common::ApiError) do |exception|
```

Comment:
@lindgrenj6 Why does this have to be ApiError, since most of the exceptions inherit from the StandardError should we be able to just use the StandardError.

Reply:
I had thought about rescuing from standard error, but didn't want to be overly rescuing. Refactored a bit and I now have 2 rescue_from for StandardError and RuntimeError